src/elements/button/accent-button/readme.md

```diff
@@ -85,7 +85,7 @@ guard against such cases in your component.
 | --------------------- | ---------------------- | ------- | ------------------------- | ------------------ | -------------------------------------------------------------------------------------------------------------------------------- |
 | `disabled`            | `disabled`             | public  | `boolean`                 | `false`            | Whether the component is disabled.                                                                                               |
 | `disabledInteractive` | `disabled-interactive` | public  | `boolean`                 | `false`            | Whether the button should be aria-disabled but stay interactive.                                                                 |
-| `form`                | `form`                 | public  | `HTMLFormElement \| null` |                    | Returns the form owner of the internals of the target element.                                                                   |
+| `form`                | `form`                 | public  | `HTMLFormElement \| null` |                    | The <form> element to associate the button with.                                                                                 |
```

Comment:
This will break in documentation, as the `<form>` will be rendered as HTML. You can escape this by wrapping `<form>` in backticks: ``` `<form>` ```